data/t_np0.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I named the original files `t0` or `t1` because t is the time variable and the number is the subscript.
`t0` is the null (control case).

Naming these files `np_t0` would be more idiomatic imo.